### PR TITLE
ixia-c release 0.0.1-2755

### PR DESCRIPTION
**Pull request by biplamal**

### CODE REVIEW

File: docs/news.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # News
 
+* **7th April, 2022**: Ixia-c version 0.0.1 (build 2755) released.  This conforms to **[v0.7.8](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.7.8/openapi.yaml)** of the Open Traffic Generator Models specification.
+
+  * This build includes following bug fix [Read more](releases.md)
+    - Clearing of `port` and `flow` statistics after `set_config`.
```
Comment (shramroy) [inline docs/news.md:6]: it should be as a part of 'set_config'

File: docs/releases.md [hunk 2]
```diff
@@ -1,6 +1,37 @@
 # Ixia-c Release Notes and Version Compatibility
 
-## Release  v0.0.1-2738 (Latest)
+## Release  v0.0.1-2755 (Latest)
+> 7th April, 2022
+
+#### About
+
+This build includes following bug fix
+- Clearing of `port` and `flow` statistics after `set_config`
```
Comment (shramroy) [inline docs/releases.md:9]: it should be as a part of 'set_config'

**Review by shramroy — CHANGES_REQUESTED** [review]
rest LGTM

File: docs/releases.md (hunk 2 above)
Comment (biplamal) [reply docs/releases.md]: Done

**Pull request merged**